sdks/python/apache_beam/internal/metrics/metric.py

```diff
@@ -262,4 +262,8 @@ def bigtable_error_code_to_grpc_status_string(grpc_status_code):
     if (grpc_status_code is not None and
         grpc_status_code in grpc_to_canonical_gcp_status):
       return grpc_to_canonical_gcp_status[grpc_status_code]
+    if grpc_status_code is None:
```

Comment:
If you move this above the previous if, you can simplify the other if by removing the `is not None` part.

Reply:
Acked, will send an update.